lib/identity_config.rb

```diff
@@ -270,6 +270,9 @@ def self.build_store(config_map)
     config.add(:lexisnexis_hmac_secret_key, type: :string)
     config.add(:lexisnexis_instant_verify_timeout, type: :float)
     config.add(:lexisnexis_instant_verify_workflow, type: :string)
+    config.add(:lexisnexis_instant_verify_workflow_ab_testing_enabled, type: :boolean)
+    config.add(:lexisnexis_instant_verify_workflow_ab_testing_percent, type: :integer)
```

Comment:
It appears to be the percentage on the alternate, but I always wonder too. Especially since there are comments sprinkled around the Acuant code saying it has no effect? We'll definitely have to check on that in the next PR.